test/Riok.Mapperly.IntegrationTests/Models/TestObject.cs

```diff
@@ -21,7 +21,9 @@ public TestObject(int ctorValue, int unknownValue = 10, int ctorValue2 = 100)
         public int IntInitOnlyValue { get; init; }
 
 #if NET7_0_OR_GREATER
-        public required int RequiredValue { get; init; }
+
+        public int RequiredValue { get; init; }
```

Comment:
Mhmmm, is this change expected?

Reply:
mb I used Codemaid for linting and it still doesn't support modern C# 😞